prometheus/collectors/expvar_collector.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collectors
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// NewExpvarCollector returns a newly allocated expvar Collector that still has
+// to be registered with a Prometheus registry.
```

Comment:
The part "that still has to be registered with a Prometheus registry" comes from the days when this was the only collector provided by the library. It has become confusing by now (because it's also true for the other collectors, but we don't say so in their doc comments). Now that it is even in its own package, we should really delete that part.